cert.go

```diff
@@ -35,7 +35,11 @@ var userAndHostname string
 func init() {
 	u, _ := user.Current()
 	if u != nil {
-		userAndHostname = u.Username + "@"
+		var n = u.Name
```

Comment:
I would name this something other than `n`

Reply:
changed